themes/openideal_theme/openideal_theme.theme

```diff
@@ -78,6 +79,18 @@ function openideal_theme_preprocess_page(&$variables) {
     || $route_match->getRouteName() == 'view.news.all_news_page') {
     $variables['is_activity_page'] = TRUE;
   }
+
+  if ($route_match->getRouteName() == 'entity.node.canonical' &&
+    $route_match->getParameter('node')->bundle() == 'idea' &&
+    \Drupal::request()->query->has('idea-duplication')) {
+    $link = Link::createFromRoute('create an idea by yourself', 'node.add', ['node_type' => 'idea'])->toString();
+    \Drupal::messenger()->addMessage(
+      t("<div>This idea might be similar to the one you were about to create.</div>
```

Comment:
Markup should not be a part of the translated text. It would be better to split each sentence and wrap the results with p tag. Please refer to any implementation of hook_help().

Reply:
Done?:)